plugins/meta/dnsname/main.go

```diff
@@ -195,10 +202,10 @@ func cmdCheck(args *skel.CmdArgs) error {
 		conffiles = append(conffiles, f.Name())
 	}
 	if !stringInSlice("addnhosts", conffiles) {
-		return errors.New("addnhost file missing from configuration")
+		return fmt.Errorf("addnhost file missing from configuration")
```

Comment:
Why not use errors.Errorf("")

Reply:
Oooh, I didn't know that existed. I only knew about `errors.New()` until now. Thanks Dan!